### ISSUE TITLE
Inception module Example in kears Doucumentation dont understand

### ISSUE THREAD

**Issue by xichie** [keras-team/keras#11195]
code bellow from keras Doucumentation Example:

from keras.layers import Conv2D, MaxPooling2D, Input

input_img = Input(shape=(256, 256, 3))

tower_1 = Conv2D(64, (1, 1), padding='same', activation='relu')(input_img)
tower_1 = Conv2D(64, (3, 3), padding='same', activation='relu')(tower_1)

tower_2 = Conv2D(64, (1, 1), padding='same', activation='relu')(input_img)
tower_2 = Conv2D(64, (5, 5), padding='same', activation='relu')(tower_2)

tower_3 = MaxPooling2D((3, 3), strides=(1, 1), padding='same')(input_img)
tower_3 = Conv2D(64, (1, 1), padding='same', activation='relu')(tower_3)

output = keras.layers.concatenate([tower_1, tower_2, tower_3], axis=1)      #axis = 1, output shape is [?, 768, 256, 64]? I think axis should be -1, and output shape is [None, 256 ,256, 192].

**Comment by tedyu** [comment]
The output is of shape (?, 768, 256, 64)

**Comment by NewQJX** [comment]
yes，you can print it.

**Comment by NewQJX** [comment]
Oh! you are right, Why not add channels?

**Comment by NewQJX** [comment]
> The output is of shape (?, 768, 256, 64)

Oh! you are right, Why not add channels?

**Comment by NewQJX** [comment]
> The output is of shape (?, 768, 256, 64)

DepthConcat used in Inception parper.

**Comment by seushengchao** [comment]
I think it should be 'axis = -1'.
